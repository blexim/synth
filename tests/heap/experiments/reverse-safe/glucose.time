WARNING! DIMACS header mismatch: wrong number of variables.
Command exited with non-zero status 10
0.14user 0.00system 0:00.15elapsed 98%CPU (0avgtext+0avgdata 16972maxresident)k
0inputs+16outputs (0major+5614minor)pagefaults 0swaps
