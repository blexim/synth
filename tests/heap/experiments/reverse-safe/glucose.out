c
c This is glucose 4.0 --  based on MiniSAT (Many thanks to MiniSAT team)
c
c ========================================[ Problem Statistics ]===========================================
c |                                                                                                       |
c |  Number of variables:         23330                                                                   |
c |  Number of clauses:           89755                                                                   |
c |  Parse time:                   0.03 s                                                                 |
c |                                                                                                       |
c | Preprocesing is fully done
c |  Eliminated clauses:           0.87 Mb                                                                |
c |  Simplification time:          0.10 s                                                                 |
c |                                                                                                       |
c ========================================[ MAGIC CONSTANTS ]==============================================
c | Constants are supposed to work well together :-)                                                      |
c | however, if you find better choices, please let us known...                                           |
c |-------------------------------------------------------------------------------------------------------|
c |                                |                                |                                     |
c | - Restarts:                    | - Reduce Clause DB:            | - Minimize Asserting:               |
c |   * LBD Queue    :     50      |   * First     :   2000         |    * size <  30                     |
c |   * Trail  Queue :   5000      |   * Inc       :    300         |    * lbd  <   6                     |
c |   * K            :   0.80      |   * Special   :   1000         |                                     |
c |   * R            :   1.40      |   * Protected :  (lbd)< 30     |                                     |
c |                                |                                |                                     |
c ==================================[ Search Statistics (every  10000 conflicts) ]=========================
c |                                                                                                       |
c |          RESTARTS           |          ORIGINAL         |              LEARNT              | Progress |
c |       NB   Blocked  Avg Cfc |    Vars  Clauses Literals |   Red   Learnts    LBD2  Removed |          |
c =========================================================================================================
c last restart ## conflicts  :  412 67 
c =========================================================================================================
c restarts              : 5 (213 conflicts in avg)
c blocked restarts      : 0 (multiple: 0) 
c last block at restart : 0
c nb ReduceDB           : 0
c nb removed Clauses    : 0
c nb learnts DL2        : 70
c nb learnts size 2     : 31
c nb learnts size 1     : 11
c conflicts             : 1069           (7244 /sec)
c decisions             : 3394           (0.00 % random) (23001 /sec)
c propagations          : 159637         (1081830 /sec)
c conflict literals     : 13441          (24.27 % deleted)
c nb reduced Clauses    : 17
c CPU time              : 0.147562 s

s SATISFIABLE
